tests/queries/0_stateless/03442_csvToJSONString.sql

```diff
@@ -0,0 +1,24 @@
+DROP TABLE IF EXISTS csv_to_json_test;
+
+CREATE TABLE
+    csv_to_json_test (entry Int32, csv String) ENGINE = Memory;
```

Comment:
Can you make this `Nullable(String)` and insert a `NULL` in there for fun?  Wanna make sure null in, null out, with no :boom: